code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -288,6 +288,7 @@
 	return
 
 /mob/living/simple_animal/attack_hand(mob/living/carbon/human/M)
+	..()

```

Comment:
I don't think we want mob buckling to mob to use a simple click to unbuckle (I'm thinking about slimes latching onto a mob, it shouldn't be easily removed I think)